CHANGELOG.md

```diff
@@ -73,6 +73,8 @@
 
 - Bumped lua-resty-session from 4.0.2 to 4.0.3
   [#10338](https://github.com/Kong/kong/pull/10338)
+- Generate fixed size keys using hashing for keys that exceed the max key size restriction in LMDB.
```

Comment:
This entry should in section `Fixes - Core`

Reply:
move